hindi/swami-vivekananda/swami-vivekananda.md

```diff
@@ -1 +1 @@
-﻿पता है स्वामी विवेकानंद जी क्या कहते थे? [यहाँ देखो|](teachings-of-swami-vivekananda/quotes.md)
+﻿पता है [स्वामी विवेकानंद](swami-vivekananda brief introduction.md) जी क्या कहते थे? [यहाँ देखो|](teachings-of-swami-vivekananda/quotes.md)
```

Comment:
This line contains more than one link.  Please put links on separate lines with a blank line in between so that options will be visually separated.